### PR TITLE
Play a Spotify URI

### PR DESCRIPTION
**Pull request by magobaol**
I have some playlists with similar names and I found that the program wasn't able to identify the right playlist I wanted to play with `play` command.
So I thought to implement a way to play any Spotify URI in order to avoid any possible name clash.

### CODE REVIEW

File: cli/commands/play.py [hunk 1]
```diff
@@ -51,40 +55,51 @@ def play(
...
-            click.echo('No results found for "{}"'.format(keyword), err=True)
-            return
-
-        item = pager.content['items'][0]
-        if play_type == 'track':
-            kwargs['data'] = {
-                'context_uri': item['album']['uri'],
-                'offset': {
-                    'uri': item['uri'],
+        if play_type == 'uri':
```
Comment (ledesmablt) [inline cli/commands/play.py]: looks like this implementation functions as expected. good one!

i think this flow here would look better and produce a nicer diff if it were structured like:
```py
if play_type == 'uri' and keyword:
    # do your new stuff here
else if keyword:
    # the original keyword handling code
```

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Commands:
   history   List your recently played tracks.
   next      Play the next track in the queue.
   pause     Pause playback.
-  play      Resume playback or search for a track/album/playlist to play.
+  play      Resume playback, play a Spotify URI or search for a track/album/playlist to play.
```
Comment (ledesmablt) [inline README.md]: think would look better as:

Resume playback, search for a track/album/playlist, or select a URI to play.

that would probably put them in order of how often they'd be used :)

**Review by ledesmablt — CHANGES_REQUESTED** [review]
requested some minor changes here, but tested it myself and this new feature works as expected. i think i'd also personally find this handy. thanks!

File: cli/commands/play.py (hunk 1 above)
Comment (magobaol) [reply cli/commands/play.py]: Much better!

File: README.md (hunk 2 above)
Comment (magobaol) [reply README.md]: You are right :-)

**Comment by magobaol** [comment]
Did the small fixes, glad you like the functionality :-)
I'll use that to quickly switch through my favourite playlists with my Stream Deck

**Review by ledesmablt — APPROVED** [review]
great!! thanks for this handy improvement.

this has me thinking about a locally stored "favorites" file that'll prioritize these provided URIs.

```bash
spotify play --uri <my playlist>
Playing: [details]

Save "My Playlist Name" to your favorites? (Y/n)

# for next time
spotify play --favorites my playlist
```

just a thought. in any case thanks for the feature man!

**Pull request merged**